client/package.json

```diff
@@ -13,32 +13,31 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^14.2.4",
-    "@angular/cdk": "^14.2.3",
-    "@angular/common": "^14.2.4",
-    "@angular/compiler": "^14.2.4",
-    "@angular/core": "^14.2.4",
-    "@angular/flex-layout": "^14.0.0-beta.40",
-    "@angular/forms": "^14.2.4",
-    "@angular/material": "^14.2.3",
-    "@angular/platform-browser": "^14.2.4",
-    "@angular/platform-browser-dynamic": "^14.2.4",
-    "@angular/router": "^14.2.4",
+    "@angular/animations": "^15.1.2",
+    "@angular/cdk": "^15.1.2",
+    "@angular/common": "^15.1.2",
+    "@angular/compiler": "^15.1.2",
+    "@angular/core": "^15.1.2",
+    "@angular/forms": "^15.1.2",
+    "@angular/material": "^15.1.2",
+    "@angular/platform-browser": "^15.1.2",
+    "@angular/platform-browser-dynamic": "^15.1.2",
+    "@angular/router": "^15.1.2",
     "rxjs": "~7.5.5",
     "tslib": "^2.4.0",
     "zone.js": "~0.11.6"
   },
   "devDependencies": {
-    "@angular-devkit/architect": "^0.1402.4",
-    "@angular-devkit/build-angular": "^14.2.4",
+    "@angular-devkit/architect": "^0.1501.3",
+    "@angular-devkit/build-angular": "^15.1.3",
     "@angular-eslint/builder": "~14.1.2",
```

Comment:
We need to update the `@angular-eslint` stuff to v15 as well, don't we?

Reply:
I upgraded this in [50ad4ad](https://github.com/UMM-CSci-3601/3601-iteration-template/pull/1280/commits/50ad4adae0c981cfe7fa9a60d0f5ccae28049c5f), so that's good now.